Disable Provides: user(NAME), group(NAME) probes as it would fire trigger on NAME uninstall

as for example uninstalling this spec: https://bugs.pld-linux.org/attachment.cgi?id=20

15:30:53  jbj> glen_: rpmns.c splits group(mailman) into the tuple {group, mailman} for name space processing.
15:32:48  jbj> the code in unbsatisfiedDepnds is driven by the name "group". if split, then its not the string
               "group(mailman)" any more.

--- rpm-4.4.9/lib/depends.c~	2008-04-08 19:44:02.000000000 +0300
+++ rpm-4.4.9/lib/depends.c	2008-04-25 15:12:52.734623679 +0300
@@ -611,6 +611,7 @@
 	goto exit;
     }
 
+#if 0
     /* Evaluate user/group lookup probes. */
     if (NSType == RPMNS_TYPE_USER) {
 	const char *s;
@@ -646,6 +647,7 @@
 	rpmdsNotify(dep, _("(group lookup)"), rc);
 	goto exit;
     }
+#endif
 
     /* Evaluate access(2) probe dependencies. */
     if (NSType == RPMNS_TYPE_ACCESS) {
--- rpm-4.4.9/lib/rpmns.c~	2007-05-16 14:31:58.000000000 +0300
+++ rpm-4.4.9/lib/rpmns.c	2008-04-25 15:36:20.223855745 +0300
@@ -179,8 +179,10 @@
     case RPMNS_TYPE_UNAME:
     case RPMNS_TYPE_SONAME:
     case RPMNS_TYPE_ACCESS:
+#if 0
     case RPMNS_TYPE_USER:
     case RPMNS_TYPE_GROUP:
+#endif
     case RPMNS_TYPE_MOUNTED:
     case RPMNS_TYPE_DISKSPACE:
     case RPMNS_TYPE_DIGEST:
